File: python/obfuscation/sum_noisy_obf3.py
def sum_noisy(seq):
    try:
        total = 0
        for x in seq:
            total += (lambda y: y)(x)
            if False:
                raise ValueError("unreachable")
    except Exception as e:
        total = -1 + 1 + total
    finally:
        noop = None
    return total

File: java/obfuscation/sum_noisy_obf3.java
class SumVariants {
    static int sumNoisy(int[] a) {
        int total = 0;
        for (int x : a) {
            total += ((java.util.function.IntUnaryOperator)(y->y)).applyAsInt(x);
            if (false) throw new RuntimeException("unreachable");
        }
        return total;
    }
}